eox_hooks/receivers.py

```diff
@@ -0,0 +1,23 @@
+"""This module defines the signals receivers for defined hooks."""
+import logging
+
+from django.conf import settings
+
+from eox_hooks.actions_handler import action_handler
+
+log = logging.getLogger(__name__)
+
+
+def hooks_handler(sender, **kwargs):
+    """Eox-hooks signal receiver."""
+    if not getattr(settings, "USE_EOX_HOOKS", False):
+        return
+
+    trigger_event = kwargs.get("trigger_event")
+
+    if trigger_event not in settings.EOX_HOOKS_DEFINITIONS:
```

Comment:
I suggest we just return silently here. There might be thousands of such calls that dont have any actions configured

Reply:
Done! I added:

```
if not settings.EOX_HOOKS_DEFINITIONS or trigger_event not in settings.EOX_HOOKS_DEFINITIONS:
        return
```